src/main/java/wooteco/subway/line/LineController.java

```diff
@@ -0,0 +1,68 @@
+package wooteco.subway.line;
+
+import java.net.URI;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import wooteco.subway.line.dto.LineRequest;
+import wooteco.subway.line.dto.LineResponse;
+
+@RestController
+@RequestMapping("/lines")
+public class LineController {
+
+    private final LineDao lineDao;
+
+    public LineController(LineDao lineDao) {
+        this.lineDao = lineDao;
+    }
+
+    @PostMapping
+    public ResponseEntity<LineResponse> createLine(@RequestBody LineRequest lineRequest) {
+        Line line = new Line(lineRequest.getName(), lineRequest.getColor());
+        Line newLine = lineDao.save(line);
+        LineResponse lineResponse = new LineResponse(newLine.getId(), newLine.getName(),
+            newLine.getColor());
+        return ResponseEntity.created(URI.create("/lines/" + newLine.getId())).body(lineResponse);
+    }
+
+    @GetMapping(produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<List<LineResponse>> showLines() {
+        List<Line> lines = lineDao.findAll();
+
+        List<LineResponse> lineResponses = lines.stream()
+            .map(it -> new LineResponse(it.getId(), it.getName(), it.getColor()))
+            .collect(Collectors.toList());
+        return ResponseEntity.ok().body(lineResponses);
+    }
+
+    @GetMapping(value = "/{id}", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<LineResponse> showLine(@PathVariable Long id) {
+        Line line = lineDao.findById(id);
+        LineResponse lineResponse = new LineResponse(line.getId(), line.getName(), line.getColor());
+        return ResponseEntity.ok().body(lineResponse);
+    }
+
+    @PutMapping("/{id}")
+    public ResponseEntity updateLine(@PathVariable Long id, @RequestBody LineRequest lineRequest) {
```

Comment:
```suggestion
    public ResponseEntity<Void> updateLine(@PathVariable Long id, @RequestBody LineRequest lineRequest) {
```

바디가 비었을 땐 `Void` 타입을 지정하시면 돼요.